start/sw/gui-app/gtk.md

```diff
@@ -0,0 +1,60 @@
+---
+title: GTK+
+subsection: gui-app
+section: start-sw
+description: 
+---
+
+# Create an application using GTK+
+
+GTK+ is a cross platform graphical toolkit which stands for GIMP toolkit.
+It was started in 1998 as an free and open source alternative to Qt which was
+proprietary at the time. GTK+ is published under the GNU LGPL 2.1 license.
+
+## Installing the development packages
+
+GTK+ is written in C so the minimum required to create a GTK+ application is a
+C compiler and the GTK+ development package:
+
+```
+$ sudo dnf install gcc gtk3-devel
+```
+
+After installing the compiler and development package you can follow the GTK+
+[getting started guide][0] on GNOME website.
+
+## Other languages
+
+GTK+ is written in C but it also have official support for C++, JavaScript, 
+Python, Vala and Rust.
+
+### Vala
+
+If you want to develop in Vala, you need to install a Vala compiler in addition
```

Comment:
Could you make a more specific description of what Vala is?

Does it possibly deserve standalone page?